main/webapp/modules/core/scripts/index/default-importing-controller/parsing-panel.html

```diff
@@ -3,7 +3,7 @@
   <table>
   <tr>
   <td width="1px"><button bind="startOverButton" class="button"></button></td>
-  <td width="1px"><button bind="previousButton" class="button"></button></td>
+  <td width="1px"><button bind="previousButton" class="button">Back</button></td>
```

Comment:
Good job! But this should be translatable, so it should not be added to this HTML file, but added to the DOM in the corresponding JS file using `i18n`.

Reply:
got it